src/components/positions.js

```diff
@@ -35,13 +25,14 @@ const Positions = () => {
     <div className="PingCAP-Positions columns is-multiline">
       {positions.map((p) => (
         <Link
-          to={`/careers/${replaceTitle(p.parent.relativePath)}`}
-          key={p.frontmatter.title}
+          to={p.link}
+          key={p.title}
           className="position column is-4"
```

Comment:
The title may be duplicated, so it shouldn't be used as a key. Using link is a better choice.